File: week02/day02/draw-triangle.py
# Write a program that reads a number from the standard input, then draws a
# triangle like this:
#
# *
# **
# ***
# ****
#
# The triangle should have as many lines as the number was
print("add a number:")
x = int(input())
for i in range(1, x+1):
    i=i*"*"
    print(i)

File: week07/day02/draw-triangle.js
'use strict';
// Write a program that draws a
// triangle like this:
//
// *
// **
// ***
// ****
//
// The triangle should have as many lines as lineCount is

var lineCount = 4;
var star = '';
for (var i = 1; i <= lineCount; i += 1) {
  star += '*';
  console.log(star);
}
